common/decisions/korea_decisions.txt

```diff
@@ -0,0 +1,181 @@
+﻿unite_korea = {
+	is_shown = {
+		NOT = {
```

Comment:
Is_shown trigger is a bit weird, right now it'll only be shown if ROK does not exist and NKR owns all of korea, and either if you have the united korea variable or own all of korea
which isnt possible at the same time

Reply:
in case it gets lost in discord messages: 

I made it so it's triggered by you or your subjects owning states, as you can see I made a failsafe, if the UN stuff goes through and ROK gets created, or if somehow ROK gets released beforehand, there's no point for the decision, and if NRK owns all states anyway then there's no point for the decision as well, that's why the NOT = { NRK = { owns_state